sdcm/logcollector.py

```diff
@@ -1067,6 +1086,12 @@ def define_test_id(self):
     def get_aws_instances_by_testid(self):
         instances = list_instances_aws({"TestId": self.test_id}, running=True)
         filtered_instances = filter_aws_instances_by_type(instances)
+        if self.params["use_cloud_manager"]:
+            if not cluster_cloud:
+                print("Could not collect siren manager logs, cluster_cloud isn't installed")
+            else:
+                filtered_instances['siren_manager_nodes'] = [cluster_cloud.get_manager_instance_by_cluster_id(
+                    cluster_id=self.params['cloud_cluster_id'])]
```

Comment:
What about following code:

```python
try:
    from cluster_cloud import get_manager_instance_by_cluster_id
except ImportError:
    LOGGER.error(...)
else:
    filtered_instances['siren_manager_nodes'] = [get_manager_instance_by_cluster_id(cluster_id=self.params['cloud_cluster_id'])]
```

?

Reply:
sound reasonable, but I need some one to try it out, and report it's working or not.